lib/src/markdown_processor.dart

```diff
@@ -318,18 +645,22 @@ String _renderMarkdownToHtml(String text, [ModelElement element]) {
 // (like, [Apple<int>]). @Hixie asked for a warning when there's something, that looks
 // like a non HTML tag (a generic?) outside of a `[]` block.
 // https://github.com/dart-lang/dartdoc/issues/1250#issuecomment-269257942
-void _showWarningsForGenericsOutsideSquareBracketsBlocks(String text,
-    [ModelElement element]) {
+void _showWarningsForGenericsOutsideSquareBracketsBlocks(
+    String text, ModelElement element) {
   List<int> tagPositions = findFreeHangingGenericsPositions(text);
   if (tagPositions.isNotEmpty) {
     tagPositions.forEach((int position) {
-      String errorMessage = "Generic type handled as HTML";
-      if (element != null) {
-        errorMessage += " in ${_elementLocation(element)}";
-      }
-      errorMessage +=
-          " - '${text.substring(max(position - 20, 0), min(position + 20, text.length))}'";
-      warning(errorMessage);
+      String priorContext =
```

Comment:
These magic constants (20, 30) are slightly disconcerting. Perhaps use named constants so why we chose their values is  more clear?